common/app/views/fragments/collections/popularExtended.scala.html

```diff
@@ -0,0 +1,86 @@
+                <ol class="tabs__container js-tabs" id="js-popular-tabs" role="tablist">
+                @popular.zipWithRowInfo.map{ case (section, info) =>
+                <li class="tabs__tab--most-popular@if(info.isFirst){ tabs__tab--selected}" role="tab" id="tabs-popular-@info.rowNum-tab"@if(info.isFirst){ aria-selected="true"} aria-controls="tabs-popular-@info.rowNum">
+                    <a href="#tabs-popular-@info.rowNum" data-link-name="tab @info.rowNum @section.heading"><span class="u-h">Most viewed </span>@Html(Localisation(section.heading.stripPrefix("popular ")))</a>
+                </li>
+                }
+                </ol>
+                <div class="tabs__content js-tabs-content">
+                }
+
+        @popular.zipWithRowInfo.map{ case (section, info) =>
+        <div id="tabs-popular-@info.rowNum"
+        class="@if(isTabbed){js-tab-@info.rowNum @if(!info.isFirst){modern-hidden}} @tabsPaneCssClass(containerDefinition) u-cf"
+            @if(isTabbed){
+                role="tabpanel"
+                aria-labelledby="tabs-popular-@info.rowNum-tab"
+            }
+        data-link-name="@section.heading"
+        data-link-context="most-read/@section.section">
+
+            <ul class="u-unstyled most-popular">
+            @section.trails.zipWithRowInfo.map{ case (trail, info) =>
+                @defining(FaciaCardHeader.fromTrail(trail, None)) { header =>
+                    <li class="most-popular__item @toneClass(trail)--most-popular fc-item--pillar-@trail.maybePillar.nameOrDefault">
+                        <div class="most-popular__link" data-link-name="@info.rowNum | text">
+                            <span class="most-popular__number most-popular__number--@info.rowNum">
+                            @fragments.inlineSvg(s"number-${info.rowNum}", "numbers")
+                            </span>
+                            <div class="most-popular__headline">
+                                @title(header, 2, 2, "headline-list__body")
+                                @trail.properties.maybeContent.map { content =>
+                                    @if(content.tags.tags.exists(_.id == "tone/news")) {
+                                        @fragments.contentAgeNotice(ContentOldAgeDescriber(content))
+                                    }
+                                }
+                            </div>
+                        </div>
+                        <a href="@header.url.get(request)" class="u-faux-block-link__overlay js-headline-text" data-link-name="article" tabindex="-1" aria-hidden="true">@RemoveOuterParaHtml(header.headline)</a>
+                    </li>
+                }
+            }
+        </ul>
+        </div>
+        }
+        @if(showMPU(containerDefinition) && isFront) {
+            <div class="fc-slice__popular-mpu fc-slice__item--mpu-candidate">
+            </div>
+        }
+        @if(isTabbed) {
+        </div>
+        </div>
+        }
```

Comment:
If you put the content of this block into a subtemplate you could avoid having the same condition twice around the opening and closing tags.
Eg. becomes something like:
```
@if(condition) {
<div>
block
</div>
} else {
block
}
```

Reply:
Agreed. I didn't want to risk refactoring the mostPop component though. I'll add that to the card that will say 'Get rid of the old article most pop version' once we're happy: https://trello.com/c/qETL0JDD/225-get-rid-of-the-old-most-pop-version